src/mappings/mappingHandlers.ts

```diff
@@ -49,6 +65,184 @@ BigInt.prototype.toJSON = function () {
   return this.toString();
 };
 
+export async function handleIbcSendPacketEvent(cosmosEvent: CosmosEvent): Promise<void> {
+  const { event, block } = cosmosEvent;
+  if (event.type != EVENT_TYPES.SEND_PACKET) {
+    logger.warn("Not valid send_packet event.");
+    return;
+  }
+
+  const packetSrcPortAttr = event.attributes.find((a) => a.key === PACKET_SRC_PORT_KEY);
+  if (!packetSrcPortAttr || packetSrcPortAttr.value !== TRANSFER_PORT_VALUE) {
+    logger.warn("packet_src_port is not transfer");
+    return;
+  }
```

Comment:
Is it possible that event attributes are not encoded? While working on indexing balances, I noticed that some events had attributes that were not encoded.

Reply:
yes it is possible. in fact, packet_src_port is not encoded